.gitignore

```diff
@@ -16,4 +16,4 @@ vgcore.*
 /dunstify
 /dunst.systemd.service
 /org.knopwob.dunst.service
-/test/test
+/test/test
```

Comment:
Please try to undo this change to not clutter the diff

Reply:
> I haven't looked at the implications of pipes and double forks, but I'm sure you can figure that out. If it's too complicated you can stick with mmap.

I'll take a look. I think I will have to basically read the output twice. Not really a problem per se. :slightly_smiling_face: 


> There are some dbus interfaces already. Most of them are accesible with `dunstctl`. You can a look at the man page of dunstctl to see what's there.

I will :slightly_smiling_face: I will be looking for Dunst event that state has changed (paused/unpaused) at the minimum and possibly notification that amount of notifications on hold has changed.


> You are able to add tests. You can add them in `test/notification.c` or any other relevant test file. Test that rely on you being able to see if the notification is drawn correctly, you can add in `test/functional_test/test.sh`.

I'll take a look at it.